base/sysinfo.jl

```diff
@@ -438,16 +438,18 @@ const WINDOWS_VISTA_VER = v"6.0"
     Sys.isexecutable(path::String)
 
 Return `true` if the given `path` has executable permissions.
+
+!!! note
+    Prior to Julia 1.4, this did not correctly interrogate filesystem
+    ACLs on Windows, therefore it would return `true` for any
+    file.  From Julia 1.4 on, it correctly determines whether the
```

Comment:
```suggestion
    file.  From Julia 1.5 on, it correctly determines whether the
```

Reply:
I think we will merge this today, but your edit still stands.  :)

Reply:
The only thing holding this up is that it's breaking the windows buildbots because the buildbots get read-only files created, and then a later process doesn't know how to delete them.  :(